tests/unittests/Foundation/NSPointerFunctionsTests.mm

```diff
@@ -80,7 +80,8 @@ NSUInteger NSPointerFunctionsTestStructSizeFunction(const void* item) {
     const void* ptr3 = (void*)((long)ptr1 + 1);
 
     testEqualFunction(functions, ptr1, ptr2, ptr3);
-    testHashFunction(functions, ptr1, ptr2, ptr3);
+    // Disabling hash function check here as performing hash on opaque values cannot be accuratly matched with the reference platform.
+    // testHashFunction(functions, ptr1, ptr2, ptr3);
```

Comment:
>// testHashFunction(functions, ptr1, ptr2, ptr3); [](start = 4, length = 49)

What was the thinking behind commenting this out, instead of a) keeping it in an #ifdef WINOBJC, or b) deleting it?